tests/integration/test_bayesian_optimization.py

```diff
@@ -229,6 +229,18 @@ def test_bayesian_optimizer_with_gpr_finds_minima_of_simple_quadratic(
     _test_optimizer_finds_minimum(GaussianProcessRegression, min(num_steps, 5), acquisition_rule)
 
 
+@random_seed
+@pytest.mark.slow
+def test_bayesian_optimizer_with_vgp_finds_minima_of_scaled_branin() -> None:
+    _test_optimizer_finds_minimum(
+        VariationalGaussianProcess,
+        10,
+        EfficientGlobalOptimization[SearchSpace, GPflowPredictor](
```

Comment:
Easiest way to remove the type ignore is to make the EGO specific to this model type. (You could also specify HasTrajectorySampler instead, but this way requires less thinking and there's not import tax.)

```suggestion
        EfficientGlobalOptimization[SearchSpace, VariationalGaussianProcess](
```